miden-tx/src/tests/mod.rs

```diff
@@ -46,23 +47,28 @@ mod kernel_tests;
 
 #[test]
 fn transaction_executor_witness() {
-    let data_store = MockDataStore::default();
+    let data_store = TransactionContextBuilder::with_standard_account(
```

Comment:
nit: here and in other places in this file `data_store` should probably be renamed to `tx_context`.